tex/chProgramming.tex

```diff
@@ -2095,15 +2178,18 @@ \section{Notations, abbreviations}\label{sec:notabrev}
 
 \begin{itemize}
 \item Concepts that are typically denoted with a letter, like $N(G)$ for the
-	normalizer of the group $G$ are represented by symbols beginning
+	normalizer of the group $G$, are represented by symbols beginning
 	with \C{'} as in \C{'N(G)}, where \C{'N} is a symbol.
 \item At the time of writing, notations for numerical constants are specially
 	handled by the system.  The algebraic part of the library overrides
 	specific cases, like binding \C{1} and \C{0} to ring elements.
 \item Postfix notations begin with \C{.}, as in \C{.+1} and \C{.-group}
 	to let one write \C{(p.-group G)}. There is one exception for
         the postfix notation of the factorial, which starts with
-        \C{`}, as in \C{m`!}.
+        \C{`}, as in \C{m`}\C{!}.
```

Comment:
Feel free to make any changes you think are helpful. You've done a very thorough review so far -- I'll just stick to commenting at the moment :)